### PR TITLE
apply null/undefined checks in website static js

### PR DESCRIPTION
**Pull request by aravind1078**
in other pages, except where they intend to have an effect these are breaking because of the values being undefined so added an if to handle those

### CODE REVIEW

**Comment by aravind1078** [comment]
> May I suggest a different approach?
> You could add a class to the `li`s you need (`.member-item`?) and after that use a code like this:
> 
> ```js
> const elemetsList = document.querySelectorAll('.member-item');
> if (elements.length > 0) {
> 	const teamArr = Array.from(elementsList);
>     // ... same logic as before
> }
> ```

yeah, we could probably do that way as well but in that case, we need to add the class to every item we put in the list...
@ematipico any particular advantage of doing the way, you are suggesting? just curious to know.

**Comment by ematipico** [comment]
Sure, I think this approach is better because:
- we won't risk to select any other `li` that are in the current page
- querying by class you express intent, meaning that you are selecting exactly `.menu-item`
- you don't know if in the future the page will contain another element that has the same structure of the `li`s that contain the one you need

Also, `Array` has a method `.shuffle` if you want to consider it

**Comment by aravind1078** [comment]
@ematipico I changed it to use the query selector... but I couldn't find any shuffle method in Array can you point to some resource?

**Comment by ematipico** [comment]
@aravind1078 sorry my band, I must have seen some other page by mistake 😅

**Comment by aravind1078** [comment]
> @aravind1078 sorry my band, I must have seen some other page by mistake 😅

oh no problem, I was searching MDN and all bcz never really heard of shuffle array method

File: website/static/js/index.js [hunk 1]
```diff
@@ -204,9 +205,11 @@ elements.overlay.addEventListener("touchstart", mobileToggleEvent, false);
 window.addEventListener("scroll", handleScroll, false);
 
 elements.colorSchemeSwitch.addEventListener("click", modeSwitch, false);
-const teamArr = Array.from(elements.teamList.getElementsByTagName("li"));
-for (const li of randomShuffle(teamArr)) {
-	elements.teamList.appendChild(li);
+const teamArr = document.querySelectorAll(".team-member-item");
```
Comment (sebmck) [inline website/static/js/index.js:208]: You can avoid having the `team-member-item` class by using a descendant selector.

```suggestion
const teamArr = document.querySelectorAll(".team-list li");
```

**Comment by aravind1078** [comment]
@sebmck  done 👍

**Comment by sebmck** [comment]
Thank you @aravind1078!

**Pull request merged**